src/Tests.Microsoft.Security.Utilities.Core/RegexPatternTests.cs

```diff
@@ -425,4 +425,25 @@ public void RegexPatterns_RaisesArgumentException_OnNullPattern()
             new RegexPattern(Id, Name, DetectionMetadata.HighEntropy, null);
         });
     }
+
+    [TestMethod]
+    public void RegexPattern_RegexOptions_UsesDefault()
```

Comment:
>RegexPattern_RegexOptions_UsesDefault [](http://example.com/codeflow?start=16&length=37)

Is this test actually passing?

If you pass a non-null default '0' won't that flow through?

Reply:
It is passing null is used here, there is no explicit arg. I will add another test for explicit none.

Reply:
Talked offline, DetectionMetadata.None looked like RegexOptions.None. Improved tests to make it more obvious.